app/public/model/PrimaryModel.js

```diff
@@ -78,6 +78,14 @@ export default class PrimaryModel extends Observable {
         this.notify();
     }
 
+    async sync() {
```

Comment:
users should not be able to start sync procedure, so such method in public model is unnecessary